src/main/java/net/openhft/chronicle/network/TcpEventHandler.java

```diff
@@ -149,6 +149,9 @@ public TcpEventHandler(@NotNull final T nc, final TcpHandlerBias bias) {
         readLog = new NetworkLog(this.sc, "read");
         writeLog = new NetworkLog(this.sc, "write");
         nbWarningEnabled = Jvm.warn().isEnabled(getClass());
+        if (nbWarningEnabled) {
```

Comment:
Not sure if this will prevent unnecessary class loading... I _think_ it should only load the class if the branch is taken?

Reply:
I am more concerned about concise code TBH